src/main/kotlin/com/ajax/cryptocurrency/service/CryptocurrencyService.kt

```diff
@@ -0,0 +1,65 @@
+package com.ajax.cryptocurrency.service
+
+import com.ajax.cryptocurrency.exception.CryptocurrencyPriceNotFoundException
+import com.ajax.cryptocurrency.exception.InvalidParamsForThePageException
+import com.ajax.cryptocurrency.model.Cryptocurrency
+import com.ajax.cryptocurrency.repository.CryptocurrencyRepository
+import org.springframework.data.domain.PageRequest
+import org.springframework.data.domain.Sort
+import org.springframework.stereotype.Service
+import java.io.File
+import java.io.FileWriter
+
+@Service
+class CryptocurrencyService(private val cryptocurrencyRepository: CryptocurrencyRepository) {
+    fun findMinMaxPriceByCryptocurrencyName(name: String, sortOrder: Int?): Cryptocurrency {
```

Comment:
`sortOrder: Int?` should be non-nullable.

Reply:
fixed